### DISCUSSION TITLE
About pycalphad

### DISCUSSION THREAD

**Discussion by Arosyisagoodcivilizen** [pycalphad/pycalphad#588] [Q&A]
1.If there is a TDB file, can the entire binary system phase diagram be drawn by modifying part of the code calculation?
2.Where can I find the function call documentation in pycalphad?

**Comment by bocklund** [comment]
1. Yes, you can see examples for plotting binary phase diagrams here: https://pycalphad.org/docs/latest/examples/BinaryExamples.html
2. API documentation is here: https://pycalphad.org/docs/latest/api/modules.html
